components/gfx/display_list/mod.rs

```diff
@@ -218,24 +219,14 @@ impl DisplayList {
     }
 }
 
-impl HeapSizeOf for DisplayList {
-    fn heap_size_of_children(&self) -> usize {
-        self.background_and_borders.heap_size_of_children() +
-            self.block_backgrounds_and_borders.heap_size_of_children() +
-            self.floats.heap_size_of_children() +
-            self.content.heap_size_of_children() +
-            self.positioned_content.heap_size_of_children() +
-            self.outlines.heap_size_of_children() +
-            self.children.heap_size_of_children()
-    }
-}
-
+#[derive(HeapSizeOf)]
 /// Represents one CSS stacking context, which may or may not have a hardware layer.
 pub struct StackingContext {
     /// The display items that make up this stacking context.
     pub display_list: Box<DisplayList>,
 
     /// The layer for this stacking context, if there is one.
+    #[ignore_heap_size = "FIXME(njn): other fields may be measured later, esp. `layer`"]
```

Comment:
It looks like this is the only unmeasured field in StackingContext now, in which case this comment should just say something like "FIXME(njn): should measure this at some point".